scripts/evaluation/collect_stats.jl

```diff
@@ -0,0 +1,56 @@
+using ArgParse
+using DrWatson
+@quickactivate
+using BSON
+using Random
+using FileIO
+using DataFrames
+using Base.Threads: @threads
+using GenerativeAD
+
+s = ArgParseSettings()
+@add_arg_table! s begin
+    "source_prefix"
+		arg_type = String
+		default = "evaluation/images"
```

Comment:
Running the collection accidentally should just update the default dataframe, which serves as a cache for the current state of computed experiments. If you intend to change those dataframes, for example group some models together, etc. I would advise you to save it in a different one, so that the default name corresponds to this raw cache.